crates/libs/wallet/src/db/mod.rs

```diff
@@ -40,6 +41,33 @@ pub const CREATE_TABLE_MELT_QUOTE: &str = r#"
             transfer_ids TEXT
         );"#;
 
+
+
+pub const CREATE_TABLE_WAD: &str = r#"
+        CREATE TABLE IF NOT EXISTS wad (
+            id INTEGER PRIMARY KEY AUTOINCREMENT,
```

Comment:
oh no sorry, I meant change the name of the column to `id`, but keep it an uuid/TEXT type